network/templates/base/observation_view.html

```diff
@@ -235,6 +235,15 @@ <h3 class="panel-title">
                   </button>
                 </a>
               {% endif %}
+              {% if data.rise_azimuth %}
+                Rise Az: {{ data.rise_azimuth }}°
+              {% endif %}
+              {% if data.max_altitude %}
+                Max Alt: {{ data.max_altitude }}°
+              {% endif %}
+              {% if data.set_azimuth %}
+                Set Az: {{ data.set_azimuth }}°
+              {% endif %}
```

Comment:
Could you wrap all these line inside a `<span class="hidden-xs"></span>`. Just to make sure we don't break the mobile view.